spec/models/report_spec.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Report, type: :model do
+  describe '#editable' do
+    let(:user) do
+      User.create!(
+        name: 'alice',
+        email: 'alice@example.com',
+        password: '123456',
+        password_confirmation: '123456'
+      )
+    end
+    let(:another_user) do
+      User.create!(
+        name: 'bob',
+        email: 'bob@example.com',
+        password: '123456',
+        password_confirmation: '123456'
+      )
+    end
+    let(:report) do
+      user.reports.create!(
+        content: 'aliceの日報です、日報を書きます。今回は編集が可能かのテストのために書きました。',
+        title: '今日のレポート'
+      )
+    end
+    context 'ユーザーがレポートを所有している' do
+      it '編集できる' do
+        expect(report.editable?(user)).to be_truthy
+      end
+    end
+    context 'ユーザーがレポートを所有していない' do
+      it '編集できない' do
+        expect(report.editable?(another_user)).to be_falsey
+      end
+    end
+  end
+  describe '#created_on' do
```

Comment:
```suggestion

  describe '#created_on' do
```

describeとdescribeの間も改行した方が見やすいです